### PR TITLE
Business Lookup on PPR Debtor Search

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#15472

*Description of changes:*
* Implements Business LookUp on for PPRs Business Debtor Search
* Minor Test updates 
* Minor Type Error fix

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/search/BusinessSearchAutocomplete.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <v-card
-    v-if="searchValue.length >= 3 && !searching"
+    v-if="searchValue && searchValue.length >= 3 && !searching"
```
Comment (owillborn) [inline ppr-ui/src/components/search/BusinessSearchAutocomplete.vue:3]: would it be possible to use something like `searchValue?.length` here instead?

Comment (cameron-eyds) [reply ppr-ui/src/components/search/BusinessSearchAutocomplete.vue:3]: That would be nice wouldn't it!
Optional Chaining is a TypeScript feature which is not supported in the Vue template itself.

There might be a way but i'm not sure if its related to our vue version or what. We've short circuited instead for some time, might be worth looking at if this is possible!

Comment (cameron-eyds) [reply ppr-ui/src/components/search/BusinessSearchAutocomplete.vue:3]: Was curious, a quick bit of research mentions support for this (typescript in the tempalte) in Vue 3 👍

Comment (owillborn) [reply ppr-ui/src/components/search/BusinessSearchAutocomplete.vue:3]: Neat! Ok thanks for the clarification

**Review by owillborn — APPROVED** [review]
LGTM

**Review by doug-lovett — APPROVED** [review]
Looks good to me.

**Pull request merged**